react/Header/UserAccount/UserAccount.less

```diff
@@ -42,19 +55,39 @@
   right: 0;
   transform-origin: 50% 0;
   transform: translateY(@grid-row-height * -3);
-  transition: opacity .25s cubic-bezier(0.020, 1.505, 0.745, 1.235), transform .25s cubic-bezier(0.020, 1.505, 0.745, 1.235);
+  transition: opacity .25s ease, transform .25s cubic-bezier(0.020, 1.505, 0.745, 1.235);
```

Comment:
Do we use these animations anywhere else yet?  Might be nice to extract them into less variables at some point.